crates/primitives-traits/Cargo.toml

```diff
@@ -19,6 +19,8 @@ alloy-eips.workspace = true
 alloy-genesis.workspace = true
 alloy-primitives.workspace = true
 alloy-rlp.workspace = true
+alloy-serde = { workspace = true }
+alloy-rpc-types = { workspace = true }
```

Comment:
alloy-rpc-types is a helper crate that just re-exports, we should use only those crates here that we really need